aips/create_dip.py

```diff
@@ -220,9 +226,15 @@ def create_dip(aip_dir, aip_uuid, output_dir):
             continue
 
         premis = techmd.contents.document
-        original_name = premis.findtext(
-            "premis:originalName", namespaces=metsrw.utils.NAMESPACES
-        )
+        # Update PREMIS namespace based on version attribute
+        premis_version = premis.get("version", "2.2")
+        try:
+            namespaces["premis"] = PREMIS_VERSIONS_NS[premis_version]
+        except KeyError:
+            LOGGER.warning(
+                "Could not update namespace for PREMIS version: %s" % premis_version
+            )
+        original_name = premis.findtext("premis:originalName", namespaces=namespaces)
```

Comment:
You may want to use `premisrw` instead because it takes care of this for you. E.g.:

```python
>>> premis_objects = fsentry.get_premis_objects()
>>> premis_objects[0].object_characteristics_extension
```

Reply:
I tried that, but `original_name` is not represented in the PREMISObject schema. I was going to create an issue to add it in there, but we also need the last modified data from FITS in here.